modules/cnv-importing-vmware-vm.adoc

```diff
@@ -1,67 +1,59 @@
+.. Select *VMware* from the *Provider* list.
+.. Select *Connect to New Instance* or a saved vCenter instance from the *vCenter instance* list.
 
-** *vCenter Hostname*
-** *vCenter User Name*
-** *vCenter Password*
-+
-The wizard connects to the vCenter instance and saves the credentials.
+.. Select a virtual machine or a template to import.
 +
-If you clear the *Save as New vCenter Instance* check box and click *Check*, the wizard connects to the vCenter instance without saving the credentials.
+The *Name* and *Operating System* are filled in automatically with the attributes of the selected virtual machine or template.
 
-* If you select a saved vCenter instance, the wizard connects to it by using the saved credentials.
+.. Select an existing flavor or *Custom* from the *Flavor* list.
+.. Select a *Workload Profile* and then click *Next*.
 
-.. Click *VM to import* and select a virtual machine or template.
-+
-The *Name* and *Operating System* are filled in automatically with the attributes of the selected virtual machine or template.
+. In the *Networking* screen, perform the following steps:
 
-.. Specify the *Memory (GB)* and number of *CPUs* if you use the default flavor, *Custom*.
+.. Click the Options menu {kebab} of a network interface and select *Edit*.
+.. Enter a valid network interface name.
 +
-Optionally, you can select an existing *Flavor*.
+The name can contain lower-case letters (`a-z`), numbers (`0-9`), and hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name must not contain upper-case letters, spaces, periods (`.`), or special characters.
 
-.. Select a *Workload Profile*.
-.. Click *Next*.
-. Perform the following procedure in the *Networking* screen:
-.. Click *NETWORK CONFIGURATION* and select *Pod Networking*.
-.. Select a *BINDING METHOD*.
-.. Click the &#10003; button to save the update.
-.. Click *Next*.
-. Perform the following procedure in the *Storage* screen:
-.. Click each disk and select a *STORAGE CLASS*. If you do not select a storage class, {CNVProductName} uses the default storage class to create the virtual machine.
+.. Select a network definition from the *Network* list.
+.. Click *Save* and then click *Next*.
 
-.. Click the &#10003; button to save the update.
-.. Select a *Bootable Disk* if you have more than one bootable disk.
-.. Click *Create Virtual Machine >*.
-+
-The *Result* screen displays the resources created for the virtual machine.
-. Click *Close*.
+. In the *Storage* screen, perform the following steps:
 
-To view the progress of the import:
+.. Click the Options menu {kebab} of a disk and select *Edit*.
++
+.. Enter a valid disk name.
++
+The name can contain lower-case letters (`a-z`), numbers (`0-9`), and hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name must not contain upper-case letters, spaces, periods (`.`), or special characters.
+.. Select a storage class.
++
```

Comment:
Before the *Storage class* there is the *Interface* field with content VirtIO for example.